lib/tasks/import.rake

```diff
@@ -3,10 +3,10 @@
 require "gds_api/json_client"
 
 namespace :import do
-  desc "Import all documents matching an organisation and document type from Whitehall Publisher, e.g. import:whitehall_migration[organisation-slug, document-type]"
-  task :whitehall_migration, %i[organisation_slug document_type] => :environment do |_, args|
+  desc "Import all documents matching an organisation and document type from Whitehall Publisher, e.g. import:whitehall_migration[organisation-content-id, document-type]"
+  task :whitehall_migration, %i[organisation_content_id document_type] => :environment do |_, args|
```

Comment:
I wonder if rather than asking users to call the rake task with a content-id, which isn't very friendly, we could change slug to base_path, and do a lookup to get the content-id? 
See: https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/publishing_api.rb#L77